internal/authorization/access_control_rule.go

```diff
@@ -32,6 +33,7 @@ func NewAccessControlRule(rule schema.ACLRule, networksMap map[string][]*net.IPN
 
 // AccessControlRule controls and represents an ACL internally.
 type AccessControlRule struct {
+	ID        int
```

Comment:
I'd call it `Position` instead of ID because my fear is that one day we put references to ACL in the DB and people might think it's a primary key whereas it's not. If we decide to agree on Position, then we should propagate through out all functions and update argument names in the function signatures.